transport/http/outbound.go

```diff
@@ -386,6 +390,9 @@ func (o *Outbound) withCoreHeaders(req *http.Request, treq *transport.Request, t
 		req.Header.Set(EncodingHeader, encoding)
 	}
 
+	// TODO(pedge): uncomment when this is supported
+	//req.Header.Set(AcceptsBothResponseErrorHeader, AcceptTrue)
```

Comment:
Let’s track this TODO. I see why you’d want to postpone the header until there’s a corresponding behavior!